apps/api-journeys/src/modules/journey/journey.spec.ts

```diff
@@ -85,13 +91,64 @@ describe('JourneyModule', () => {
         const journey = {
           id: uuidv4(),
           title: 'published',
-          published: true,
           locale: 'hi-IN',
           themeName: ThemeName.base,
           themeMode: ThemeMode.light,
           description: null,
           primaryImageBlockId: null,
-          slug: 'published-slug'
+          slug: 'published-slug',
+          publishedAt,
+          createdAt
+        }
+        dbMock.journey.findUnique.mockResolvedValue(journey)
+
+        const { data } = await query(
+          gql`
+            query ($id: ID!) {
+              journey(id: $id) {
+                id
+                title
+                locale
+                themeName
+                themeMode
+                slug
+                publishedAt
+                createdAt
+              }
+            }
+          `,
+          {
+            id: journey.id
+          }
+        )
+
+        expect(data?.journey).toEqual(
+          pick(journey, [
+            'id',
+            'title',
+            'published',
+            'locale',
+            'themeName',
+            'themeMode',
+            'slug',
+            'publishedAt',
+            'createdAt'
+          ])
+        )
+      })
+
+      it('publishedAt prop is null if journey is not yet published', async () => {
```

Comment:
```suggestion
      it('should not be published by default', async () => {
```

Also add tests:
`should set createdAt with current date time` - just check the date and time to the hour is correct.
`should set publishedAt when _____` - trigger the published condition and check the date/time like above.